apps/cookbook/src/app/showcase/list-load-on-demand-showcase/list-load-on-demand-showcase.component.ts

```diff
@@ -7,6 +9,5 @@ declare var require: any;
   styleUrls: ['./list-load-on-demand-showcase.component.scss'],
 })
 export class ListLoadOnDemandShowcaseComponent {
-  exampleHtml: string = require('!raw-loader!../../examples/list/components/load-on-demand/list-load-on-demand-example.component.html')
-    .default;
+  exampleHtml: string = config.template;
```

Comment:
Consider aligning with other showcase examples (get the template from the example instance in the markup)